### DISCUSSION TITLE
Linux: Can I change HOMEBREW_PREFIX="/home/linuxbrew/.linuxbrew"

### DISCUSSION THREAD

**Discussion by lamyergeier** [Homebrew/discussions#4076] [Linux] [answered]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

In [install/install.sh at master · Homebrew/install](https://github.com/Homebrew/install/blob/master/install.sh), can I change `HOMEBREW_PREFIX="/home/linuxbrew/.linuxbrew"` to any other path that I want such as `/mnt/Data/opt/linuxbrew/.linuxbrew`?

### Output of `brew doctor`

```shell
-
```

### Description of issue

Change of prefix for homebrew installation

**Comment by SMillerDev** [comment, marked as answer]
No, you need the untar method in https://docs.brew.sh/Installation for that.
